index.js

```diff
@@ -16,33 +25,47 @@ module.exports = compose
  * @api public
  */
 
-function compose (middleware) {
+const compose = (middleware) => {
+  const isProduction = process.env.NODE_ENV === 'production'
+  if (isProduction) return composeSlim(middleware)
```

Comment:
Doing so won't make tree shaking happen. You need to do:
```suggestion
  if (process.env.NODE_ENV === 'production') return composeSlim(middleware)
```